features/iframe-credentialless.yml

```
@@ -0,0 +1,7 @@
+name: Iframe credentialless
+description: The `credentialless` attribute for `<iframe>` loads third-party content in an ephemeral context and does not send any credentials such as cookies. When using cross-origin isolation, this allows you to embed content that do not send COEP headers.
```

A couple style nits here:

- "the `<foo>` HTML element" for consistency with other descriptions
- Subject-verb agreement
- (perhaps controversially) expanding "COEP" to what it stands for

```suggestion
description: The `credentialless` attribute for the `<iframe>` HTML element loads third-party content in an ephemeral context and does not send any credentials such as cookies. When using cross-origin isolation, this allows you to embed content that does not send `Cross-Origin-Embedder-Policy` headers.
```